financial-connections/src/main/java/com/stripe/android/financialconnections/domain/ConfirmVerification.kt

```diff
@@ -13,21 +14,64 @@ internal class ConfirmVerification @Inject constructor(
         consumerSessionClientSecret: String,
         verificationCode: String,
     ): ConsumerSession = requireNotNull(
```

Comment:
do you still need `requireNotNull` ?

Reply:
https://github.com/stripe/stripe-android/pull/6570/commits/d4fed0d6672ea1f72116a2dabf789fd07e60f2da